shared/common-adapters/button.desktop.js

```diff
@@ -144,66 +144,65 @@ const buttonCommon = {
   minWidth: 10
 }
 
-const styles = {
```

Comment:
as mentioned in person let's move away from having this style object. eslint doesn't help us find orphaned unreferenced properties in here